skietbaan_fe/src/bootstrap/LeaderboardStyle.css

```diff
@@ -67,22 +90,21 @@
    text-align: center
 }
 .curentMember{
-    background-color: #171F32 !important;background-image: linear-gradient(-90deg, #FF611D, #171F32);
-    color: white !important;
-    position: fixed;
+    color: #fdfdfd !important;
+    /* position: fixed; */
     bottom: 0;
     left: 0;
     width: 100%;
     padding: 0%;
+    background-color: #e11e2b;
 }
-.curentMember table{
-    background-image: linear-gradient(-90deg, #FF611D, #171F32);
+.curentMember *{
+    background-color: #e11e2b !important;
+    color: #fdfdfd !important;
 }
-.curentMember table td{
-    padding: 0% !important;
-}
-.filters{
-    background-image: linear-gradient(-90deg, #FF611D, #171F32);
+.rankRows .tr .td{
+    padding-top: 2% !important;
```

Comment:
You are making use of `!important` a lot which should not be a habit. Rather find the source of the problem and write more specific css classes.

Reply:
There are 53 uses of `!important` in this file alone. That is way too much. If you are struggling with css ask for help but abusing the use of `!important` will make the code unmaintainable, messy and cause a lot of unpredictable behaviour.